test/service/test_response.py

```diff
@@ -956,6 +965,28 @@ def test_response_factory_projects_cell_line(self):
         samples = one(one(hits)['samples'])
         self.assertElasticEqual(samples, expected_samples)
 
+    def test_response_factory_projects_bionetworks(self):
+        hits = self._get_hits('projects', 'c765e3f9-7cfc-4501-8832-79e5f7abd321')
+        factory = SearchResponseFactory(hits=hits,
+                                        pagination=self.paginations[0],
+                                        aggs={},
+                                        entity_type='projects',
+                                        catalog=self.catalog)
+        response = factory.make_response()
+        hits = response['hits']
+        expected_tissue_atlas = [
+            {'atlas': None, 'version': None},
+            {'atlas': None, 'version': None},
+            {'atlas': 'Lung', 'version': None},
+            {'atlas': 'Retina', 'version': 'v1.0'},
+            {'atlas': 'Blood', 'version': 'v1.0'},
+        ]
+        expected_bionetwork_name = ['Eye', 'Immune', 'Kidney', 'Lung', 'Skin']
+        project = one(one(hits)['projects'])
+        self.assertEqual(expected_tissue_atlas, project['tissueAtlas'])
+        self.assertEqual(expected_bionetwork_name, project['bionetworkName'])
+        self.assertTrue(project['isTissueAtlasProject'])
```

Comment:
These "factory" tests were grandfathered in from when I took the project over. We shouldn't add more because they only provide limited coverage of service code. I'd prefer a test that actually hits the service endpoint. Ideally, you would just add add assertions to an existing test method. You may need to modify a different bundle to get there.